test/modules/standard/FileSystem/lydia/symlink/SKIPIF

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Try to make a link.  If it fails, don't do this test.
+
+ln -s foo.txt blahblahblah.txt
```

Comment:
I don't think you need this skipif.

Our test systems are capable of creating links everywhere. We have issue with symlinks that are checked in; some of the test systems cannot create the symlinks correctly when git cloning.